InformationScripting/scripts/color.py

```diff
@@ -8,6 +8,6 @@
 for ts in inputs:
     tuples = ts.take("ast")
     for tuple in tuples:
-        tuple.add(NamedProperty("color", color))
-        ts.add(tuple)
+        t = Tuple([NamedProperty("color", color), NamedProperty("ast", tuple.ast)])
```

Comment:
Hmm, this is a bit verbose. Isn't there a way to get rid of the two `NamedProperties` on this line and somehow make those implicit?